app/javascript/dashboard/routes/dashboard/conversation/conversation.routes.js

```diff
@@ -15,12 +15,12 @@ export default {
       },
     },
     {
-      path: frontendURL('accounts/:accountId/inbox-view/:conversation_id'),
+      path: frontendURL('accounts/:accountId/inbox-view/:notification_id'),
```

Comment:
@pranavrajs @muhsin-k I will implement this as a child route rather than handling it here using if cases. And I will move it out from the conversation routes in a separate PR. 

This PR mostly includes
1. Adds the ability to snooze notifications from the context menu,
2. Used notification ID instead of conversation ID for routes (Added it in the PR because to use notification ID 3. directly to snooze notification in detail view instead of using finding logic)
3. Removed end of list message (Small changes)
4. Close the inbox list display and options menu if the inbox card context menu is open. (Small changes)
5. And some changes that needed when used notification ID instead of conversation ID for routes